### PR TITLE
note on transaction units, bones to HNT conversion ratio

### PR DESCRIPTION
**Pull request by samgutentag**
resolution for issue #715

### CODE REVIEW

**Comment by samgutentag** [comment]
Screen Shot 2022-06-10 at 5 58 22 PM
Screen Shot 2022-06-10 at 5 58 57 PM

File: docs/api/blockchain/accounts.mdx [hunk 1]
```diff
@@ -956,6 +956,13 @@ _Query Parameters_
 
 200: OK
 
+:::note Transaction Units
+
+Remember that transaction units are in `bones`. Read more
+[here](/api/blockchain/introduction/#transaction-units)
+
+:::
+
```
Comment (Perronef5) [inline docs/api/blockchain/accounts.mdx:965]:
```suggestion
import TransactionUnitsSnippet from "snippets/transaction-units-snippet.mdx"

<TransactionUnitsSnippet />
```

Comment (Perronef5) [reply docs/api/blockchain/accounts.mdx:965]: Just a suggestion but this should make it easier to maintain in case we want to change this in the future

**Review by Perronef5 — APPROVED** [review]
Looks good just a small suggestion

**Pull request merged**